Core/Frameworks/Flake/Core/Database.php

```diff
@@ -29,10 +29,10 @@
 
 abstract class Database extends \Flake\Core\FLObject {
 
-    private $debugOutput;
-    private $debug_lastBuiltQuery;
-    private $store_lastBuiltQuery;
-    private $oDb;
+    protected $debugOutput;
+    protected $debug_lastBuiltQuery;
+    protected $store_lastBuiltQuery;
+    protected $oDb;
```

Comment:
Sorry about that. `phpstan` was complaining about these vars not being declared in the abstract class.
I made them `private` and hadn't looked through the class that inherit from this to see that they were `protected`